debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/TestHelper.java

```diff
@@ -326,7 +326,7 @@ protected static void assertNoOpenTransactions() throws SQLException {
 
             try {
                 Awaitility.await()
-                        .atMost(TestHelper.waitTimeForRecords(), TimeUnit.SECONDS)
+                        .atMost(TestHelper.waitTimeForRecords() * 5, TimeUnit.SECONDS)
```

Comment:
Would not it be just enough to increase the wait time in the system property?

Reply:
It would, but we've also used this approach in quite a number of places too so I think either would work so I can change it however you'd prefer.  Would you prefer adjusting the system property instead?

Reply:
My bad, you are right, this is a special case only so it makes sense to multiply it by a constant. When CI completes the PR can be merged.